utils/time.ts

```diff
@@ -0,0 +1,71 @@
+import dayjs, { Dayjs } from 'dayjs';
+import utc from 'dayjs/plugin/utc'; // dependent on utc plugin
+import timezone from 'dayjs/plugin/timezone';
+import { ProgramView_events } from '../__generated__/ProgramView_events.graphql';
+import { ArrayElement } from './types';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+export const timeIsBetween = (
+  time: Dayjs,
+  start: string,
+  end: string,
+  date: string
```

Comment:
should this also be of type Dayjs? and the times as well? And why not just pass in a dayjs datetime object instead of both time and date for both times?

Reply:
Sure, we could do the parsing in the components